mrmustard/lab/detectors.py

```diff
@@ -214,9 +219,7 @@ def recompute_project_onto(
 
 
 class Heterodyne(Parametrized, GaussianMeasurement):
-    r"""
-    Heterodyne measurement on a given mode.
-    """
+    r"""Heterodyne measurement on a given mode."""
```

Comment:
Should move the args to this docstring instead (don't forget to remove them below).
```suggestion
    r"""Heterodyne measurement on a given mode.
    
    Args:
        mode: modes of the measurement
        x: x-coordinates of the measurement
        y: y-coordinates of the measurement
    """
```